gstreamer-rtsp-server/src/auto/r_t_s_p_server.rs

```diff
@@ -0,0 +1,328 @@
+// This file was generated by gir (37f7805+) from gir-files (???)
+// DO NOT EDIT
+
+use RTSPAuth;
+use RTSPClient;
+use RTSPMountPoints;
+use RTSPSessionPool;
+use RTSPThreadPool;
+use ffi;
+use glib;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::signal::SignalHandlerId;
+use glib::signal::connect;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::boxed::Box as Box_;
+use std::mem;
+use std::mem::transmute;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct RTSPServer(Object<ffi::GstRTSPServer, ffi::GstRTSPServerClass>);
+
+    match fn {
+        get_type => || ffi::gst_rtsp_server_get_type(),
+    }
+}
+
+impl RTSPServer {
+    pub fn new() -> RTSPServer {
+        assert_initialized_main_thread!();
+        unsafe {
+            from_glib_full(ffi::gst_rtsp_server_new())
+        }
+    }
+
+    //pub fn io_func(socket: /*Ignored*/&gio::Socket, condition: glib::IOCondition, server: &RTSPServer) -> bool {
+    //    unsafe { TODO: call ffi::gst_rtsp_server_io_func() }
+    //}
+}
+
+impl Default for RTSPServer {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+unsafe impl Send for RTSPServer {}
+unsafe impl Sync for RTSPServer {}
+
+pub trait RTSPServerExt {
+    fn attach<'a, P: Into<Option<&'a glib::MainContext>>>(&self, context: P) -> u32;
+
+    //fn client_filter<'a, P: Into<Option<&'a /*Unimplemented*/RTSPServerClientFilterFunc>>, Q: Into<Option</*Unimplemented*/Fundamental: Pointer>>>(&self, func: P, user_data: Q) -> Vec<RTSPClient>;
+
+    //fn create_socket<'a, P: Into<Option<&'a /*Ignored*/gio::Cancellable>>>(&self, cancellable: P, error: /*Ignored*/Option<Error>) -> /*Ignored*/Option<gio::Socket>;
+
+    //fn create_source<'a, P: Into<Option<&'a /*Ignored*/gio::Cancellable>>>(&self, cancellable: P, error: /*Ignored*/Option<Error>) -> Option<glib::Source>;
```

Comment:
Needs manual implementation, see other things that return sources elsewhere. But different PR